maestral/main.py

```diff
@@ -1280,28 +1280,15 @@ def shutdown_daemon(self) -> None:
         """
 
         self.stop_sync()
-        self._cleanup_aio_tasks()
+
+        self._refresh_task.cancel()
+        self._thread_pool.shutdown(wait=False)
 
         if self._loop.is_running():
             self._loop.call_soon_threadsafe(self.shutdown_complete.set_result, True)
 
     # ==== private methods ===============================================================
 
-    def _cleanup_aio_tasks(self):
```

Comment:
Not sure what this is and how it got here.

Reply:
I think this is from a rebase of the develop branch a few days ago. Could you rebase this PR to the develop branch and resolve any conflicts in `maestral/daemon.py` and `maestral/main.py` in favour of the develop branch?

Reply:
I rebased against develop, one of the test steps is complaining, but I guess that's expected.

Reply:
The Docker build job works as expected: https://github.com/aries1980/maestral/runs/1204192542?check_suite_focus=true